python/ray/tests/BUILD

```diff
@@ -288,14 +288,20 @@ py_test_module_list(
   files = [
     "test_gcs_ha_e2e.py",
     "test_gcs_ha_e2e_2.py",
-    "test_memory_pressure.py",
     "test_node_labels.py",
   ],
   size = "medium",
   tags = ["exclusive", "team:core", "xcommit"],
   deps = ["//:ray_lib", ":conftest"],
 )
 
+py_test_module_list(
+  files = ["test_memory_pressure.py"],
```

Comment:
maybe lump it together with other medium tests, instead of having its separate job? I think it was remained in civ1 because we thought that civ2 caused the test to be flaky. Probably no longer the case and @rkooo567 fixed the test flakiness recently I think.

Reply:
I think the issue with this test is that it does not work with other tests, or it at least does not work when running in parallel. it seems to have very tight assumption on how the system's memory is being used.

even when it is running on its own but in your `rayci` wrapper in a container, it becomes flaky.